resources/src/commands/introduction_reply.rs

```diff
@@ -0,0 +1,35 @@
+use crate::slack::add_reaction;
+use crate::slack::chat_post_message;
+
+pub async fn reply_to_introduction(
```

Comment:
Since the base mod is `introduction_reply` you can probably just name this method send or post since the context is already there on what we are sending when the method is accessed.